modules/administration/pages/kubernetes.adoc

```diff
@@ -1,175 +1,173 @@
 [[kubernetes]]
 = Kubernetes
 
+You can use {productname} to manage {k8s} containerized clients in addition to regular traditional or Salt clients.
 
+{productname} manages {k8s} clusters with a virtual host manager.
```

Comment:
I'm not sure if "manage" is the right word here. SUMA doesn't manage k8s clusters per se. The whole purpose of this integration is to read info from the cluster and report if their SUMA-built containers are outdated, and do CVE audits on them.

Reply:
I'd also prefer "clusters" to "containerized clients"